### PR TITLE
Update .prettierignore adding client/js/dist and .venv

### PR DESCRIPTION
**Pull request by whitphx**
# Description

These directories should also be ignored by Prettier when developing the frontend.

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have added a short summary of my change to the CHANGELOG.md
- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

# A note about the CHANGELOG

Hello 👋 and thank you for contributing to Gradio!

All pull requests must update the change log located in CHANGELOG.md, unless the pull request is labeled with the "no-changelog-update" label.

Please add a brief summary of the change to the Upcoming Release > Full Changelog section of the CHANGELOG.md file and include
a link to the PR (formatted in markdown) and a link to your github profile (if you like). For example, "* Added a cool new feature by `[@myusername](link-to-your-github-profile)` in `[PR 11111](https://github.com/gradio-app/gradio/pull/11111)`".

If you would like to elaborate on your change further, feel free to include a longer explanation in the other sections.
If you would like an image/gif/video showcasing your feature, it may be best to edit the CHANGELOG file using the 
GitHub web UI since that lets you upload files directly via drag-and-drop.

### CODE REVIEW

File: .config/.prettierignore [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 **/js/app/dist/**
+**/client/js/dist/**
 **/pnpm-lock.yaml
 **/js/plot/src/Plot.svelte
 **/.svelte-kit/**
 **/demo/**
 **/gradio/**
 **/website/**
-**/venv/**
+**/.venv/**
```
Comment (whitphx) [inline .config/.prettierignore:11]: The `venv` directory [configured in `.gitignore`](https://github.com/gradio-app/gradio/blob/3227a9b703fb520bf4f8a8ee08f9027c0a5c1070/.gitignore#L53) is `.venv`.

**Review by pngwn — COMMENTED** [review]
Thanks for this @whitphx! Just a small comment about the changelog.

File: CHANGELOG.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ No changes to highlight.
 
 ## Full Changelog:
 
-No changes to highlight.
+* Updated `.prettierignore` by [@whitphx](https://github.com/whitphx) in [PR 4160](https://github.com/gradio-app/gradio/pull/4160)"
```
[the remark this answers is not in the document]
Comment (whitphx) [reply CHANGELOG.md:28]: It makes sense, but the CI flow `check-changelog / Check CHANGELOG file` fails which is required to merge if we don't update the changelog.

[the remark this answers is not in the document]
Comment (abidlabs) [reply CHANGELOG.md:28]: I added the `no-changelog-update` label so we should be good to remove this entry @whitphx

**Review by pngwn — APPROVED** [review]

**Pull request merged**